source/docs/articles/drupal.md

```diff
@@ -1,6 +1,43 @@
 ---
 title: Drupal
-description: All about Drupal, on Pantheon.
+description: All about Drupal on Pantheon.
 category: drupal
-draft: true
+keywords: drupal, sites, pantheon, upstream
 ---
+[Drupal](https://www.drupal.org) is an open source content management platform powering millions of websites and applications. It’s built, used, and supported by an active and diverse community of people around the world.
+
+Pantheon is the only hosting platform hyper-optimized for both Drupal and WordPress. Because we optimize our entire infrastructure for websites we deliver the best [performance available](http://getlevelten.com/blog/kyle-taylor/best-bang-no-bucks-comparison-free-drupal-hosting-platforms).
```

Comment:
Let's replace this paragraph with information about the drops-7 upstream. (link to the repo, explain relationship to pressflow-7).